public/v4/apps/meilisearch.yml

```diff
@@ -0,0 +1,113 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: getmeili/meilisearch:$$cap_app_version
+        volumes:
+            - $$cap_appname-data:/$$cap_db_path
+            - $$cap_appname-snapshot:/$$cap_snapshot_dir
+            - $$cap_appname-dump:/$$cap_dump_dir
```

Comment:
Defining the "inside container" paths as customizable variables usually provide little value to the user. 

Users don't really care about what path is used inside the container. Let's remove these and just hardcode these:
- `$$cap_db_path`
- `$$cap_snapshot_dir`
- `$$cap_dump_dir`

Reply:
Thanks for the review.
Committed the required changes.